src/pages/index.js

```diff
@@ -0,0 +1,307 @@
+import "./index.css";
+
+import { resetValidation, enableValidation, settings, disableButton } from "../scripts/validation.js";
+import { setButtonText } from "../utils/helpers.js";
+import Api from "../utils/Api.js";
+
+const initialCards = [
+  {name: "Val Thorens", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg"},
+  {name: "Restaurant terrace", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg"},
+  {name: "An outdoor cafe", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg"},
+  {name: "A very long bridge, over the forest", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg"},
+  {name: "Tunnel with morning light", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg"},
+  {name: "Mountain house", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg"},
+  {name: "Golden Gate Bridge", link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg"},
+];
+
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ea7dc041-453a-47d1-92f9-2ec7e3393acf",
+    "Content-Type": "application/json"
+  }
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+    if (userInfo && userInfo.name && userInfo.about) {
+      const avatarImage = document.querySelector("#avatar");
+      avatarImage.src = userInfo.avatar;
+
+      const userName = document.querySelector(".profile__name");
+      userName.textContent = userInfo.name;
+
+      const userTitle = document.querySelector(".profile__description");
```

Comment:
`document.querySelector(".profile__description")`

You've already found it in line 55 as `profileDescription`. Please, use it here